src/arrays/programs_involving_data_sequences/easy/q8/MainTest.java

```diff
@@ -0,0 +1,93 @@
+package arrays.programs_involving_data_sequences.easy.q8;
+
+import global.BaseTest;
+import global.MethodTest;
+import global.utils.ArrayUtil;
+import global.variables.Clause;
+import global.variables.clauses.IntegerLiteral;
+import global.variables.clauses.NewLine;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
```

Comment:
Formatt this code please

Reply:
I did but it didn't change anything, what exactly doesn't look to be formatted?